docs-chef-io/content/inspec/resources/ibmdb2_conf.md

```diff
@@ -0,0 +1,58 @@
++++
+title = "ibmdb2_conf resource"
+draft = false
+gh_repo = "inspec"
+platform = "os"
+
+[menu]
+  [menu.inspec]
+    title = "ibmdb2_conf"
+    identifier = "inspec/resources/os/ibmdb2_conf.md ibmdb2_conf resource"
+    parent = "inspec/resources/os"
++++
+
+Use the `ibmdb2_conf` Chef InSpec audit resource to test the configuration settings. Make sure you are using the IBM Db2 database instance user credentials to run the InSpec test.
+
+## Availability
+
+### Installation
+
+This resource is distributed along with Chef InSpec itself. You can use it automatically.
+
+## Syntax
+
+A `ibmdb2_conf` resource block declares db2_executable_file_path, db_instance to connect and then runs command to get the configuration values and compares it to the value stated in the test:
+
+describe ibmdb2_conf(db2_executable_file_path: "/opt/ibm/db2/V11.5/bin/db2", db_instance: "db2inst1") do
+  its("output") { should_not be_empty }
+  its("output") { should include("Audit buffer size (4KB) (AUDIT_BUF_SZ) = 0")}
+end
+
+Windows
+
+describe ibmdb2_conf do
+  its("output") { should_not be_empty }
+  its("output") { should include("Audit buffer size (4KB) (AUDIT_BUF_SZ) = 0")}
+end
+
```

Comment:
Is there a way to do this with `db_name`?

For example:

```ruby
describe ibmdb2_conf(db_name: "NAME") do
    test things
end

Reply:
Its not required in case of Windows.

Reply:
Is `db_name` required for linux?

Reply:
for ibmdb2_conf resource, db_name is not required for both Linux and windows.